src/vaults/RefundEscrow.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.28;
+
+import {IRefundEscrow} from "./interfaces/IRefundEscrow.sol";
+
+import {Auth} from "../misc/Auth.sol";
+
+contract RefundEscrow is Auth, IRefundEscrow {
+    constructor(address owner) Auth(owner) {}
+
+    receive() external payable {}
+
+    function depositFunds() external payable auth {}
+
+    function withdrawFunds(address to, uint256 value) external auth {
+        (bool success,) = to.call{value: value}("");
```

Comment:
AFAICT, this would fail with an unclear error if `value > balance`. 
```suggestion
        require(address(this).balance >= value, InsufficientBalance());
        (bool success,) = to.call{value: value}("");
```

Reply:
The caller ensures that this never happen, already passing `address(refund).balance`, so the error will never be dispatched